### PR TITLE
updated tutorials

### PR DESCRIPTION
**Pull request by davitbzh**

### CODE REVIEW

File: docs/getting_started/quickstart.ipynb [hunk 1]
```diff
@@ -765,7 +772,9 @@
    "source": [
     "## 🤖 Create Deployment\n",
     "\n",
-    "Next step is to deploy the model on KServe behind Hopsworks for real-time inference requests."
+    "Next step is to deploy the model on KServe behind Hopsworks for real-time inference requests.\n",
+    "\n",
+    "**Note:** deployments for scikit-learn without Kserve require a predictor scrip. For details please refer to [Hopsworks documentation](https://docs.hopsworks.ai/latest/user_guides/mlops/serving/predictor/#step-2-optional-implement-predictor-script).  "
```
Comment (MagicLex) [inline docs/getting_started/quickstart.ipynb:777]: "**Note:** deployments for scikit-learn without Kserve requires a predictor script. For details please refer to [Hopsworks documentation](https://docs.hopsworks.ai/latest/user_guides/mlops/serving/predictor/#step-2-optional-implement-predictor-script).  "

**Review by MagicLex — APPROVED** [review]

**Review by moritzmeister — DISMISSED** [review]

File: docs/tutorials/fraud_online/1_feature_groups.ipynb [hunk 2]
```diff
@@ -2,25 +2,15 @@
...
-  },
-  {
-   "cell_type": "markdown",
-   "id": "86f66405",
-   "metadata": {},
-   "source": [
-    "# Part 01: Load, Engineer & Connect\n",
-    "\n",
-    "[![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/logicalclocks/hopsworks-tutorials/blob/master/fraud_online/1_feature_groups.ipynb)\n",
+    "# <span style=\"font-width:bold; font-size: 3rem; color:#1EB182;\"><img src=\"../images/icon102.png\" width=\"38px\"></img> **Hopsworks Feature Store** </span><span style=\"font-width:bold; font-size: 3rem; color:#333;\">- Part 01: Load, Engineer & Connect</span>\n",
```
Comment (moritzmeister) [inline docs/tutorials/fraud_online/1_feature_groups.ipynb:8]: Pretty sure this won't work. The image doesn't exist here. Also why all this HTML?

File: docs/tutorials/fraud_online/1_feature_groups.ipynb [hunk 3]
```diff
@@ -2,25 +2,15 @@
...
     "\n",
-    "This is the first part of the quick start series of tutorials about Hopsworks Feature Store. As part of this first module, we will work with data related to credit card transactions. \n",
-    "The objective of this tutorial is to demonstrate how to work with the **Hopworks Feature Store**  for batch data with a goal of training and deploying a model that can predict fraudulent transactions.\n",
+    "<span style=\"font-width:bold; font-size: 1.4rem;\"> This is the first part of the quick start series of tutorials about Hopsworks Feature Store. As part of this first module, we will work with data related to credit card transactions. \n",
+    "The objective of this tutorial is to demonstrate how to work with the **Hopworks Feature Store**  for batch data with a goal of training and deploying a model that can predict fraudulent transactions.</span>\n",
     "\n",
-    "## 🗒️ This notebook is divided in 3 sections:\n",
+    "## **🗒️ This notebook is divided in 3 sections:** \n",
```
Comment (moritzmeister) [inline docs/tutorials/fraud_online/1_feature_groups.ipynb:13]: The stars are redundant, they don't do anything if it's already a title.

File: docs/tutorials/fraud_online/1_feature_groups.ipynb [hunk 4]
```diff
@@ -2,25 +2,15 @@
...
+    "# <span style=\"font-width:bold; font-size: 3rem; color:#1EB182;\"><img src=\"../images/icon102.png\" width=\"38px\"></img> **Hopsworks Feature Store** </span><span style=\"font-width:bold; font-size: 3rem; color:#333;\">- Part 01: Load, Engineer & Connect</span>\n",
     "\n",
-    "This is the first part of the quick start series of tutorials about Hopsworks Feature Store. As part of this first module, we will work with data related to credit card transactions. \n",
-    "The objective of this tutorial is to demonstrate how to work with the **Hopworks Feature Store**  for batch data with a goal of training and deploying a model that can predict fraudulent transactions.\n",
+    "<span style=\"font-width:bold; font-size: 1.4rem;\"> This is the first part of the quick start series of tutorials about Hopsworks Feature Store. As part of this first module, we will work with data related to credit card transactions. \n",
+    "The objective of this tutorial is to demonstrate how to work with the **Hopworks Feature Store**  for batch data with a goal of training and deploying a model that can predict fraudulent transactions.</span>\n",
```
Comment (moritzmeister) [inline docs/tutorials/fraud_online/1_feature_groups.ipynb:11]: Pretty sure the span doesn't do anything.

File: docs/tutorials/fraud_online/2_feature_view_creation.ipynb [hunk 5]
```diff
@@ -4,27 +4,27 @@
...
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "metadata": {},
-   "source": [
-    "# Part 02: Training Data & Feature views\n",
+    "# <span style=\"font-width:bold; font-size: 3rem; color:#1EB182;\"><img src=\"../images/icon102.png\" width=\"38px\"></img> **Hopsworks Feature Store** </span><span style=\"font-width:bold; font-size: 3rem; color:#333;\">- Part 02: Training Data & Feature views</span>\n",
     "\n",
-    "[![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/logicalclocks/hopsworks-tutorials/blob/master/fraud_online/2_feature_view_creation.ipynb)\n",
```
Comment (moritzmeister) [inline docs/tutorials/fraud_online/2_feature_view_creation.ipynb:16]: Why are you removing the Open in Colab buttons?

**Review by moritzmeister — CHANGES_REQUESTED** [review]
I purposefully removed all the <span> etc from the notebooks, so they get displayed properly in the docs. When I tested it, these html parts didn't change anything. Why are you adding them back?

Also you are removing the "Open in Colab" buttons.

**Comment by moritzmeister** [comment]
Added automation to pull notebooks automatically from hopsworks-tutorials

**Pull request closed without merging**